src/renderer/components/dock/logs/view.tsx

```diff
@@ -34,6 +34,12 @@ interface Dependencies {
 const NonInjectedLogsDockTab = observer(({ className, tab, model, subscribeStores }: Dependencies & LogsDockTabProps) => {
   const logListElement = createRef<LogList>();
   const data = model.logTabData.get();
+  const [tabIds, setTabIds] = useState([]);
+
+  if (!tabIds.find(id => tab.id === id)) {
+    setTabIds([...tabIds, tab.id]);
+    model.reloadLogs();
```

Comment:
Is it ok to call this here?

Reply:
I wonder if having the reloadLogs effect have a dependency on `tabId` would be enough.

Or barring that also having the `subscribeStores` effect change its dependency to `tabId`

Reply:
adding tabId as a dependency just limits when the effect is applied? As it is it should always be applied, right?